backend/codeception.yml

```diff
@@ -8,3 +8,8 @@ paths:
 settings:
     bootstrap: _bootstrap.php
     colors: true
+    memory_limit: 1024M
+modules:
+    config:
+        Yii2:
+            configFile: 'config/test-local.php'
```

Comment:
Could you elaborate what merge you mean?
I added these lines for consistency with the counter-part in  frontend/codeception.yml